imports/plugins/included/braintree/lib/collections/schemas/braintree.js

```diff
@@ -43,7 +43,8 @@ export const BraintreePayment = new SimpleSchema({
   },
   cardNumber: {
     type: String,
-    min: 16,
+    min: 12,
+    max: 19,
```

Comment:
What CC uses 19 digits? Just curious. Is this something that should be changed across all payment methods?